app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -240,18 +255,19 @@ from being executed. We are planning to remove this limitation in the future.
 
     Test the lambda function from the AWS console and make sure the execution succeeds.
 
-4. Finally we set up a Service and Route in Kong and link it to the function just created.
+4. Finally, we set up a Service and Route in Kong and link it to the
+`Mylambda` function we just created.
 
-{% tabs %}
-{% tab With a database %}
+{% navtabs %}
+{% navtab With a database %}
 
 ```bash
 curl -i -X POST http://{kong_hostname}:8001/services \
 --data 'name=lambda1' \
 --data 'url=http://localhost:8000' \
 ```
 
-The Service doesn't really need a real `url` since we are not going to have an HTTP call to upstream but rather a response generated by our function.
+The Service doesn't really need a real `url` because we're not going to have an HTTP call to an Upstream but rather a response generated by our function.
```

Comment:
```suggestion
The Service doesn't really need a real `url` because this example won't have an HTTP call to an Upstream but rather a response generated by the lambda function.
```